### DISCUSSION TITLE
What is ChistaDATA SLA?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#13] [❓FAQ] [answered]
What is ChistaDATA SLA?

**Comment by TheraniAA** [comment, marked as answer]
ChistaDATA SLA guarantees a certain level of service for its customers, including uptime and response times. The specific terms of the SLA vary depending on the service plan chosen described below but typically it includes a high level of uptime and fast response times for support requests.

Free Tier - No SLA - Community Support (Forum)
Startup Tier - Ticket Support - 4 Hours SLA
Enterprise and Critical Tier - Ticket Support - 30 mins SLA
